### PR TITLE
Some block methods and param names

### PR DESCRIPTION
**Pull request by i509VCB**

### CODE REVIEW

File: mappings/net/minecraft/block/AbstractRailBlock.mapping [hunk 1]
```diff
@@ -5,6 +5,20 @@ CLASS net/minecraft/class_2241 net/minecraft/block/AbstractRailBlock
...
+		ARG 1 state
+		ARG 2 world
+		ARG 3 pos
+		ARG 4 notify
+	METHOD method_27935 shouldDropRail (Lnet/minecraft/class_2338;Lnet/minecraft/class_1937;Lnet/minecraft/class_2768;)Z
+		COMMENT Checks if this rail should be dropped.
+		COMMENT
+		COMMENT <p>This method will return true if:
+		COMMENT <ul>The rail block is ascending.</ul>
+		COMMENT <ul>The block in the direction of ascension does not have a top rim.</ul>
```
Comment (liach) [inline mappings/net/minecraft/block/AbstractRailBlock.mapping]:
```suggestion
		COMMENT <ul><li>The rail block is ascending.</li>
		COMMENT <li>The block in the direction of ascension does not have a top rim.</li></ul>
```
html is evil indeed

File: mappings/net/minecraft/world/Difficulty.mapping [hunk 2]
```diff
@@ -7,6 +7,7 @@ CLASS net/minecraft/class_1267 net/minecraft/world/Difficulty
 		ARG 4 name
 	METHOD method_16691 byName (Ljava/lang/String;)Lnet/minecraft/class_1267;
 		ARG 0 name
+	METHOD method_27297 getNextDifficulty ()Lnet/minecraft/class_1267;
```
Comment (liach) [inline mappings/net/minecraft/world/Difficulty.mapping]: Is this `cycle` or `higher` (next higher)? or just `next` (to me this looks like getting the higher difficulty)?

Comment (i509VCB) [reply mappings/net/minecraft/world/Difficulty.mapping]: It seems to cycle as follows:

```
int newOrdinal = (oldOrdinal + 1) % Difficulty.values().length
```

So cycle probably make sense there

Comment (liach) [reply mappings/net/minecraft/world/Difficulty.mapping]:
```suggestion
	METHOD method_27297 cycle ()Lnet/minecraft/class_1267;
```

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/block/entity/LockableContainerBlockEntity.mapping [hunk 3]
```diff
@@ -11,3 +11,5 @@ CLASS net/minecraft/class_2624 net/minecraft/block/entity/LockableContainerBlock
 		ARG 1 player
 	METHOD method_17823 getContainerName ()Lnet/minecraft/class_2561;
 	METHOD method_5465 createContainer (ILnet/minecraft/class_1661;)Lnet/minecraft/class_1703;
```
Comment (Juuxel) [inline mappings/net/minecraft/block/entity/LockableContainerBlockEntity.mapping]:
```suggestion
	METHOD method_5465 createScreenHandler (ILnet/minecraft/class_1661;)Lnet/minecraft/class_1703;
```

**Review by Juuxel — CHANGES_REQUESTED** [review]

**Review by YanisBft — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**